CHANGELOG-1.5.md

```diff
@@ -1,3 +1,9 @@
+# v1.5.5
+
+### Bug Fixes
+
+- Prevent flushing custom iptable rules frequently ([474]https://github.com/Azure/aad-pod-identity/pull/474)
```

Comment:
The markup seems reverse. it not showing up as link...

Reply:
fixed!